CoreScriptsRoot/Modules/Chat.lua

```diff
@@ -1911,7 +1911,7 @@ local function CreateChatWindowWidget(settings)
 
 	local function CreateChatWindow()
 		-- This really shouldn't be a button, but it is currently needed for VR.
-		local container = Util.Create (InputService.VREnabled and 'TextButton' or 'Frame')
+		local container = Util.Create 'TextButton'
```

Comment:
Although it really should be a frame, For VR it's needed to be a button for the MouseButton1Click event. From my understanding, the reason the check didn't work is because VREnabled doesn't get set immediately when the game starts. So it would have to initialize as a frame, and then when VR does get enabled, change the entire hierarchy of the chat ui to make the base container a TextButton.